.circleci/config.yml

```diff
@@ -0,0 +1,89 @@
+---
+# all-purpose workspace persister
+persist_to_workspace: &persist_to_workspace
+  persist_to_workspace:
+    root: .
+    paths:
+      - ./
+
+version: 2.1
+
+# consolidate
+executors:
+  python-executor:
+    working_directory: ~/pantheon-systems/fusedav
+    docker:
+      - image: circleci/python:2.7.15
+  ruby-executor:
+    working_directory: ~/pantheon-systems/fusedav
+    docker:
+      - image: circleci/ruby:2.4
+
+jobs:
+  build-test:
+    executor: python-executor
+    parallelism: 1
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Install cpplint
+          command: sudo pip install cpplint
+      - run:
+          name: Set up version
+          command: bash scripts/version.sh
+      - run:
+         name: Call cpplint; expected to fail, so set things up to pass
+         command: |
+           set +eo pipefail
+           /usr/local/bin/cpplint --filter=-whitespace/line_length src/*
+           true
+      - run:
+         name: Build the image
+         command: bash scripts/docker-outer.sh
+      - *persist_to_workspace
+
+  deploy-dev:
```

Comment:
You may want to use our deploy-toolbox image. Not sure if the fusedav handling of packagecloud is and should be different from normal.

Reply:
all seems fine otherwise if the only thing to fix is the volume deletion error.